src/Main.java

```diff
@@ -6,6 +15,50 @@
     public static void main(String[] args) {
         HelloWorld hello = new HelloWorld();
         hello.say();
+
+        Scanner read = new Scanner(System.in);
+
+        while (true) {
+
+            URL website = null;
+            String url = read.nextLine();
+            try {
+                website = new URL(url);
+            } catch (MalformedURLException e) {
+                e.printStackTrace();
+            }
+
+            String html = "";
+
+            try {
+                String line;
+                BufferedReader htmlReader = new BufferedReader(new InputStreamReader(website.openStream()));
+                while ((line = htmlReader.readLine()) != null) {
+                    html += line;
+                    html += "\n";
+                }
+                htmlReader.close();
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+
+            Pattern pat = Pattern.compile("<title>.*</title>");
+            Matcher mat = pat.matcher(html);
+
+
+            if (mat.find()) {
+                String name = html.substring(mat.start()+7, mat.end()-8);
```

Comment:
if you would declare pattern as
Pattern.compile("<title>(.*)</title>")
you can then obtain name as:
String name = pat.matcher(html).group();